src/main/resources/templates/db_scripts/schema.json

```diff
@@ -31,33 +28,13 @@
         }
       ],
       "ginIndex": [
-        {
```

Comment:
Is it removed because new RMB version provides it out of the box?

Reply:
Unless I am mis-reading this, according to https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-271 

> Remove each foreign key field index and each primary key field id index...

Reply:
Ok, thanks for providing the doc.

I think the `userId` and `itemId` have not to be removed. The item and user tables are actually in another DB schemes (diku_mod_inventory_storage and diku_mod_users) so we don't have FKs for them.

Note: usually FKs are declared in schema.json under `tables.foreignKeys` (for example: https://github.com/folio-org/mod-inventory-storage/blob/master/src/main/resources/templates/db_scripts/schema.json#L318).

Reply:
Ok, I was reading it as `(foreign key field index) and (each primary key field id index)`, which from what I gather should instead be read as `foreign key (field index and each primary key field id index)`? (note the parenthesis inteded to communicate logical grouping.)

Those ginIndex are indexes for columns that act as a primary key.
If I am understanding correctly, then only the foreign key group associated with an index/pkey should be removed?

Reply:
I think it means following:
* remove all the indexes for primary key (it is index for `id`, I think) - the reason of this I think that RMB now automatically creates such index. 
* remove indexes for properties that have declared in `foreignKeys` section - again, it seems RMB now creates indexes for them automatically.

I don't think that we have to update properties that act as a FK but don't have associated FK constraint.

Reply:
We need an index for "userId" and "itemId", but not as an "ginIndex" but as a simple btree index ("index" section).
Do not remove the lines, for "ginIndex" change "tOps": "ADD" to "tOps": "DELETE" so that the indexes get removed.

Reply:
@julianladisch I thought there was a change near the end of 2019 Q4 where RAML Module Builder dropped all indexes during an upgrade and recreated only the remaining ones?

Or have I mis-recalled that situation, and a different path was chosen?

Reply:
RMB drops all indexes listed in schema.json and then recreates them because most of them were changed for the 600 char length limit.
RMB does not touch indexes not listed in the indexes sections of schema.json. Module authors may use customSnippetPath to create special indexes.

Reply:
@julianladisch Thank you for clarifying that distinction, it helps my understanding of the behaviour